src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTabController.js

```diff
@@ -11,6 +11,30 @@
      */
     onNewBatchClick: function(component, event, helper) {
         helper.checkFieldPermissions(component);
+    },
+
+    loadMoreData: function (component, event, helper) {
+        event.getSource().set("v.isLoading", true);
+        if (component.get('v.batchData').length >= component.get('v.totalNumberOfRows')) {
```

Comment:
Are we only setting totalNumberOfRows on init()? Is there an issue if second user adds one or more batch while the first user is viewing the page, it might hide batches because of this - and possibly even reveal the batch that was added while hiding older ones?

Reply:
This might be a use case for storable actions? https://developer.salesforce.com/blogs/developer-relations/2017/03/lightning-components-best-practices-caching-data-storable-actions.html